dwarfs-fuse-mac 0.10.2 (new formula)

mhx opened the pull request.

gromgit commented:
Thanks for the PR, @mhx, and apologies for the delay. I've just made several significant changes in the main branch, so can you rebase and force-push your PR branch again?

mhx commented:
Good to see you @gromgit! :)

I've rebased, updated the formula (there's been another release in the meantime that gets rid of the inline patch), and just tested the formula locally.
